src/vos/vos_container.c

```diff
@@ -230,6 +232,10 @@ cont_insert(struct vos_container *cont, struct d_uuid *key, struct d_uuid *pkey,
 		D_GOTO(exit, rc);
 	}
 
+	cont->vc_open_count = 1;
```

Comment:
I think we should set it to 1 in vos_cont_open()? 
```
rc = cont_insert(...)
if (rc)
     ...
cont->vc_open_cont = 1;

Reply:
OK